### ISSUE TITLE
Github action isn't finishing.

### ISSUE THREAD

**Issue by GMetaxakis** [r0adkll/upload-google-play#172]
**Describe the bug**
Github action isn't finishing.

The upload is happening without any issue, but the github action runs forever.

**Workflow Step Configuration**

```yml
      - name: 'GPlay deployment'
        uses: r0adkll/upload-google-play@v1
        with:
          serviceAccountJsonPlainText: ${{ secrets.SERVICE_ACCOUNT_JSON }}
          packageName: package
          releaseFiles: app/build/outputs/bundle/release/production.aab
          mappingFile: app/build/outputs/mapping/release/mapping.txt
          track: alpha
          status: draft
```

**Step Debugging**

- [ ] I have enabled [Step Debug Logging](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/enabling-debug-logging#enabling-step-debug-logging)

**Comment by Navil** [comment]
I have the same issue. I canceled my pipeline after 20 minutes and according to the logs, everything finished successfully.

**Comment by GMetaxakis** [comment]
If it is google console API issue at least if we could add a timeout ourselves [here](https://github.com/r0adkll/upload-google-play/blob/9a5e7fcdd507f75178243375c12ce7c6ea6d62d0/src/main.ts#L69) it will be nice to not waste github action minutes like that :)

**Comment by boswelja** [comment]
> If it is google console API issue at least if we could add a timeout ourselves [here](https://github.com/r0adkll/upload-google-play/blob/9a5e7fcdd507f75178243375c12ce7c6ea6d62d0/src/main.ts#L69) it will be nice to not waste github action minutes like that :) 

I did just add a timeout there, but I guess I'm misunderstanding something with the API if it just hangs

**Comment by boswelja** [comment]
Fixed by #173, release coming soon
